templates/Entry/entries.html.twig

```diff
@@ -86,7 +111,8 @@
     {% endif %}
 
     <!-- Export -->
-    <div id="export" class="sidenav">
+    {% if has_exports %}
```

Comment:
this is a non critical bugfix as before the panel was always displayed even when no export optin enabled